hooks/useNewSarsCov2Data.tsx

```diff
@@ -0,0 +1,45 @@
+import { useQuery } from "@tanstack/react-query";
+import { gql } from "@apollo/client";
+import { request } from 'graphql-request';
+import { SarsCov2EstimatesQuery } from "@/gql/graphql";
+
+export const sarsCov2Esimates = gql`
+  query sarsCov2Estimates {
+    sarsCov2Estimates {
+      antibodies
+      isotypes
+      isWHOUnityAligned
+      testType
+      sourceType
+      riskOfBias
+      populationGroup
+      sex
+      ageGroup
+      country
+      countryAlphaTwoCode
+      countryAlphaThreeCode
+      whoRegion
+      unRegion
+      gbdSuperRegion
+      gbdSubRegion
+      state
+      city
+      id
+      latitude
+      longitude
+      samplingStartDate
+      samplingEndDate
+      samplingMidDate
+      countryPeopleVaccinatedPerHundred
+      countryPeopleFullyVaccinatedPerHundred
+      countryPositiveCasesPerMillionPeople
+    }
+  }
+`
+
+export function useNewSarsCov2Data() {
```

Comment:
The name `useSarsCov2Data` is already in use. I'll make this name better once nothing is using that old hook.